python/cuml/dask/metrics/confusion_matrix.py

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import cupy as cp
+from cuml.dask.common import extract_arr_partitions
+
+from cuml.utils.memory_utils import with_cupy_rmm
+from cuml.dask.metrics.utils import sorted_unique_labels
+from cuml.prims.label import make_monotonic
+from dask.distributed import default_client
+
+
+@with_cupy_rmm
+def local_cm(inputs, labels, use_sample_weight):
```

Comment:
```suggestion
def _local_cm(inputs, labels, use_sample_weight):
```
internal function should start with `_` preferrably

Reply:
Good point!